app/controllers/interviews_controller.rb

```diff
@@ -13,6 +13,7 @@ def new
 
   def show
     @interview = Interview.find(params[:id])
+    @questions = @interview.template.questions
```

Comment:
may be not relaterd to current task
but `Interview` should have relation with `Question`